### ISSUE TITLE
Missing "loading" image in high-contrast mode

### ISSUE THREAD

**Issue by brockfanning** [open-sdg/open-sdg#4]
When loading an indicator page in high-contrast mode, an asset can't be found: `/assets/img/high-contrast/loading.gif`

We should add this asset if still needed, or otherwise remove the link to it.
